File: integration-tests/gradle/src/main/resources/custom-jar-classifier-dependency/runner/src/main/java/io/example/ExampleResource.java
package io.example;

import io.blob.Intermediate;
import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;

@Path("/hello")
public class ExampleResource {

  @GET
  @Produces(MediaType.TEXT_PLAIN)
  public String hello() {
    Intermediate intermediate = new Intermediate();
    intermediate.someMethod();
    return "Hello from Quarkus REST";
  }
}

File: integration-tests/gradle/src/main/resources/included-kotlin-build/src/main/kotlin/org/example/ExampleResource.kt
package org.example

import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.Produces
import jakarta.ws.rs.core.MediaType

@Path("/hello")
class ExampleResource(val someBean: SomeBean) {

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun hello() = someBean.someMethod()
}